packages/mattermost-redux/src/actions/channels.ts

```diff
@@ -539,6 +539,40 @@ export function fetchMyChannelsAndMembers(teamId: string): ActionFunc {
     };
 }
 
+export function fetchAllMyTeamsChannelsAndChannelMembers(): ActionFunc {
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        let channels;
+        let channelsMembers;
+        const state = getState();
+        const {currentUserId} = state.entities.users;
```

Comment:
nit: I would always prefer defining const values first

```suggestion
        const state = getState();
        const {currentUserId} = state.entities.users;
        let channels;
        let channelsMembers;
```